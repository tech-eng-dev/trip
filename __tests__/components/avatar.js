import React from 'react';
import renderer from 'react-test-renderer';
import Avatar from '../../src/components/avatar';


test('Avatar renders correctly', () => {
  const avatar = renderer.create(<Avatar
    width={60}
    userId="7MnU3xgql7O5RduyY2TpNig4KtA2"
    image={undefined}
    blocked
  />).toJSON();
  expect(avatar).toMatchSnapshot();
});
