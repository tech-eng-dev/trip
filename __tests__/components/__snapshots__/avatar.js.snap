// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F5FAFE",
      "borderRadius": 30,
      "height": 60,
      "overflow": "hidden",
      "position": "relative",
      "width": 60,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/resources/images/person.png",
      }
    }
    style={
      Object {
        "backgroundColor": "#F5FAFE",
        "borderRadius": 30,
        "height": 60,
        "overflow": "hidden",
        "position": "relative",
        "width": 60,
      }
    }
  />
  <Image
    source={
      Object {
        "testUri": "../../../src/resources/images/block.png",
      }
    }
    style={
      Object {
        "backgroundColor": "transparent",
        "borderRadius": 30,
        "height": 60,
        "opacity": 0.5,
        "overflow": "hidden",
        "position": "absolute",
        "width": 60,
      }
    }
  />
</View>
`;
